### ISSUE TITLE
Volume HUD not shown when changing through AirPods

### ISSUE THREAD

**Issue by weisJ** [Wouter01/MediaMate-Releases#39]
**Describe the bug**
I noticed that the volume HUD does not show when I change the volume through my AirPods. If I have the audio tab in the settings open (which of course forces the HUD to show) then the HUD indicator changes if I change the volume through my headphones. Is this a known limitation?

**To Reproduce**
1. Connect AirPods (not sure about other headphones)
2. Change volume through AirPods
3. Observe how the HUD isn't visible

**Expected behavior**
The HUD is visible while changing volume through headphones.

**Please complete the following information:**
 - OS: Sonoma 14.3.1
 - Device M2 Macbook Pro
 - Version 3.1.2 build 258

**Comment by Wouter01** [comment]
Have you tried enabling the "Show on External Volume Event" option?

**Comment by weisJ** [comment]
Yes I have it enabled. I think I remember this working when I first installed it a few versions ago.

**Comment by Wouter01** [comment]
Could it be possible that the airpods were also connected to another device (e.g. iPhone) and sending the volume change to that device?

I currently do not have airpods which can change the volume of my mac, which makes testing this difficult.

I have retested the "Show on external event" feature and it seems to work fine in my case, but it's with a regular bluetooth speaker.

**Comment by weisJ** [comment]
The airpods aren't connected to a different device. The system volume is adjusted it simply doesn't cause the HUD to show. I don't see this as a major issue as I usually change volume using the keyboard either way. Simply wanted to note the issue.

I will also test it with a regular bluetooth speaker to rule out any other installed programs to cause the issue.

**Comment by weisJ** [comment]
Just tested with a regular Bluetooth speaker with the same result. The hud doesn’t show when changing the volume through the speaker.

**Comment by Wouter01** [comment]
Thank you for the update. What option do you have the event controller mode set to for audio? (mediamate settings -> general -> event controller mode)

**Comment by weisJ** [comment]
It is currently set to keyboard. I have also tried System and BetterDisplay (which I have installed). System also doesn't work. BetterDisplay makes the HUD not appear at all.

**Comment by Wouter01** [comment]
Kinda puzzled what the issue could be. In "System" mode, changing the volume through a bluetooth speaker / airpods should be handled the same way as adjusting the volume through the keyboard, as everything in that mode is handled by macOS, which reports it back to MediaMate. 

As the issue isn't a major issue, I will leave it for now, and see if I get any other reports of the issue not working.

**Comment by weisJ** [comment]
Is there a way to clear all settings. I tried reinstalling but I guess the settings are stored somewhere (maybe iCloud?). My installation may be a bit borked.

On another note I have noticed that "Two Finger Drag to Change Value" is also broken for me. 

And when using the IOS Hud hovering over it while it disappears used to reshow it. This works for the classic HUD for me.

**Comment by Wouter01** [comment]
You can reset MediaMate settings by opening the settings window, and clicking the "Restore to defaults" menu bar option

![image](https://github.com/Wouter01/MediaMate-Releases/assets/62355975/379f2ee5-b3f2-418d-95ee-8445ce4379cb)

As for the two finger drag, it should be fixed in the upcoming update

**Comment by Awesomeplayer165** [comment]
I can also attest that it doesn't show up.

**Comment by weisJ** [comment]
Can confirm that two finger drag works in the latest version.

**Comment by weisJ** [comment]
Just noticed that with the latest version on macOS 26 the hud now reliably shows when changing the volume through AirPods.
